mappings/net/minecraft/tag/ItemTags.mapping

```diff
@@ -1,6 +1,11 @@
 CLASS yz net/minecraft/tag/ItemTags
+	CLASS yz$a AutoUpdatingTag
```

Comment:
`CachingTag`

Reply:
Auto updating is more important here as these tags aren't reloaded with tag manager but still can reflect latest tag contents.

Reply:
Yes, so it's caching the data and only updating it from the tag manager when necessary. That's the definition of what caching is.

Reply:
Done.